src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -49,6 +49,11 @@ enums:
         meaning: rdfs:Datatype
       rdf property:
         meaning: rdf:Property
+      sssom literal:
```

Comment:
I just realized that the `predicate_type` slot is also using the `entity_type` enum. The spec for `sssom literal` (or `rdfs literal` if we chose to reuse that value) must make clear that it can only be used for the `subject_type` and `object_type` slot. We don’t want to allow the use of “literal predicates” (what would a literal predicate even be?).

Reply:
predicate_type needs its own enum.. Grrrrr. You are of course right.. Again

Reply:
Not sure that a new enum for `predicate_type` would be necessary. If we have one enum for the subject and object entities and one enum for the predicate, there would be shared values between the two: `owl object property`, `owl data property`, `owl annotation property`, and `rdf property` were added to the `entity_type` enum primarily, I suppose, to be used to indicate the type of the predicate (in `predicate_type`), but they can also legitimately be used to indicate the type of the subject/object, if you are mapping properties (we do that for example in Uberon when we use a SSSOM set to replace some old properties by their modern RO equivalents).

So instead of having two enums with redundant values, we could simply say that only a subset of the `entity_type` values (the 4 listed above) can be used in the `predicate_type` slot. `owl class`, `owl named individual`, `skos concept`, etc. and `sssom literal` MUST NOT be used in `predicate_type`.

Reply:
This is solved as far as `rdfs literal` is concerned (the in-schema documentation explicitly states that `rdfs literal` cannot be used in `predicate_type`).

However we want to solve the issue for the other values in `entity_type` (including, possibly, by creating a new distinct enum specifically for `predicate_type`), this has no longer anything to do with literal mappings and would belong to a new, dedicated ticket.